manifests/kustomize/env/platform-agnostic-kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
You'll need to sync up with #11684 as this requires the manifests to leverage cert-manager for the webhook.